widgets/events/src/EventsTable.tsx

```diff
@@ -74,6 +74,11 @@ export default class EventsTable extends React.Component {
         return toolbox.refresh(fetchParams);
     };
 
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    isOneElementLike = (collection: object | string | null | undefined): boolean => {
```

Comment:
Minor:

We could do

```suggestion
    isOneElementLike = (collection: unknown): boolean => {
```

to be more general :slightly_smiling_face: